FetchArtifacts Exception - rewrite user auth flow

```
java -jar ./flank.jar -v
v4.2.0
5148c339e697778a3714bfb1aa6d4a9a120c5164
```

There is a failure in the end of test.

```
FetchArtifacts
  Exception in thread "main" picocli.CommandLine$ExecutionException: Error while running command (ftl.cli.firebase.test.ios.IosRunCommand@767599a7): com.google.cloud.storage.StorageException: java.lang.IllegalStateException: OAuth2Credentials instance does not support refreshing the access token. An instance with a new access token should be used, or a derived type that supports refreshing.
	at picocli.CommandLine.execute(CommandLine.java:1168)
	at picocli.CommandLine.access$800(CommandLine.java:141)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:1367)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:1335)
	at picocli.CommandLine$AbstractParseResultHandler.handleParseResult(CommandLine.java:1243)
	at picocli.CommandLine.parseWithHandlers(CommandLine.java:1526)
	at picocli.CommandLine.run(CommandLine.java:1974)
	at picocli.CommandLine.run(CommandLine.java:1920)
	at ftl.Main$Companion.main(Main.kt:48)
	at ftl.Main.main(Main.kt)
Caused by: com.google.cloud.storage.StorageException: java.lang.IllegalStateException: OAuth2Credentials instance does not support refreshing the access token. An instance with a new access token should be used, or a derived type that supports refreshing.
```

--- a/test_runner/src/main/kotlin/ftl/gc/GcStorage.kt
+++ b/test_runner/src/main/kotlin/ftl/gc/GcStorage.kt
@@ -28,7 +28,9 @@ object GcStorage {
         val builder = StorageOptions.newBuilder()
         if (FtlConstants.useMock) builder.setHost(FtlConstants.localhost)
 
-        builder.setCredentials(FtlConstants.googleCredentials).build().service
+        // The oauth lib for user auth needs to be replaced
+        // https://github.com/TestArmada/flank/issues/464#issuecomment-455227703
+        // builder.setCredentials(FtlConstants.googleCredentials)
 
         builder.build()
     }

